tests/kuttl/tests/octavia_topology/01-assert-octavia.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: octavia.openstack.org/v1beta1
```

Comment:
I might need to expand this to double check the `lastAppliedTopology` field in the `OctaviaAPI` status (and any other involved resource).